manning/manning-proposal-2020-07.md

```diff
@@ -140,7 +138,7 @@ or go into data science rather than telling readers
 how to build software
 (e.g., Zhang's upcoming *A Tour of Data Science*).
```

Comment:
Should we provide the link for *A Tour of Data Science*?
https://www.routledge.com/A-Tour-of-Data-Science-Learn-R-and-Python-in-Parallel/Zhang/p/book/9780367895860

Reply:
done